src/mesh/mesh_base.C

```diff
@@ -2148,6 +2182,11 @@ MeshBase::copy_constraint_rows(const SparseMatrix<T> & constraint_operator)
 
       Elem * added_elem = this->add_elem(std::move(elem));
       this->_elem_dims.insert(0);
+      this->_elem_default_orders.insert(added_elem->default_order());
+      this->_max_nodal_order =
+        static_cast<Order>
+          (std::min(static_cast<int>(this->_max_nodal_order),
+                    static_cast<int>(added_elem->max_nodal_order())));
```

Comment:
Since `_max_nodal_order` actually the *minimum* max nodal order, should we maybe name that parameter differently? I know that this is explained in the parameter comments, but I think it is an important distinction that could be easy to miss.